<ui version="4.0" >
 <class>EldoraCappi</class>
 <widget class="QDialog" name="EldoraCappi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QFrame" name="cappiFrame" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>3</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>600</width>
         <height>600</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_4" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ColorBar" name="colorBar" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>13</hsizetype>
           <vsizetype>13</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>40</width>
           <height>500</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Click to set scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame_2" >
          <property name="geometry" >
           <rect>
            <x>720</x>
            <y>680</y>
            <width>61</width>
            <height>61</height>
           </rect>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QFrame" name="frame_3" >
           <property name="geometry" >
            <rect>
             <x>730</x>
             <y>690</y>
             <width>61</width>
             <height>61</height>
            </rect>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string/>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="gridColorButton" >
            <property name="minimumSize" >
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Select the color for the grid</string>
            </property>
            <property name="text" >
             <string>Ring/grid color</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="backgroundButton" >
            <property name="minimumSize" >
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Select the background color</string>
            </property>
            <property name="text" >
             <string>Background color</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveImage" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Save the display to a file</string>
            </property>
            <property name="text" >
             <string>Save Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="gridCheck" >
          <property name="minimumSize" >
           <size>
            <width>1</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Enable Grid overlay</string>
          </property>
          <property name="text" >
           <string>Grid</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="autoSaveCheck" >
          <property name="minimumSize" >
           <size>
            <width>1</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Enable Grid overlay</string>
          </property>
          <property name="text" >
           <string>Image Auto Save</string>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>101</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="timeBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string/>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="panButton" >
          <property name="toolTip" >
           <string>Mouse will be used for panning</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="zoomButton" >
          <property name="toolTip" >
           <string>Mouse will be used for zooming</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string/>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QPushButton" name="zoomIn" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>1</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip" >
               <string>Zoom in both displays</string>
              </property>
              <property name="text" >
               <string>Zoom in</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="zoomOut" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>1</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip" >
               <string>Zoom out for displays</string>
              </property>
              <property name="text" >
               <string>Zoom out</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="2" column="1" >
             <widget class="QPushButton" name="panDown" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip" >
               <string>Pan both displays down</string>
              </property>
              <property name="text" >
               <string>v</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QPushButton" name="viewReset" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip" >
               <string>Reset zoom and pan on both displays</string>
              </property>
              <property name="text" >
               <string>o</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QPushButton" name="panLeft" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip" >
               <string>Pan both dsplays left</string>
              </property>
              <property name="text" >
               <string>&lt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" >
             <widget class="QPushButton" name="panRight" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip" >
               <string>Pan both displays right</string>
              </property>
              <property name="text" >
               <string>></string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QPushButton" name="panUp" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip" >
               <string>Pan both displays up</string>
              </property>
              <property name="text" >
               <string>^</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>144</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Vitals</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Grid</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="gridText" >
            <property name="toolTip" >
             <string>Grid spacing (deg)</string>
            </property>
            <property name="text" >
             <string>0.0</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Lat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="latText" >
            <property name="toolTip" >
             <string>Current latitude (deg)</string>
            </property>
            <property name="text" >
             <string>0.0</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Lon</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lonText" >
            <property name="toolTip" >
             <string>Current longitude (deg)</string>
            </property>
            <property name="text" >
             <string>0.0</string>
            </property>
            <property name="textFormat" >
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorBar</class>
   <extends>QWidget</extends>
   <header>ColorBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
